utils/compare_results.py

```diff
@@ -46,6 +46,32 @@ def get_wining_status(cls, current_status: str, proposed: str) -> str:
         return proposed
 
 
+class Comparison:
+    same_status = list()
+    missing_in_target = list()
+    different_results = dict()
+
+    def __init__(self, base_results, target_results):
+        self.base_results = base_results
+        self.target_results = target_results
+
+    def do_compare(self):
```

Comment:
Do you think that it would make sense to declare this as private and also call it from init? There is not much of useful stuff one could do with a `Comparison` that is just initialized and not processed.